jsBasics/homeWork6/task3.js

```diff
@@ -0,0 +1,33 @@
+function divideNumbers (numerator,denominator){
+    if (typeof numerator !== 'number' || typeof denominator !== 'number'){
+        throw new Error("Error: both parameters must be number!")
+    } else if(denominator === 0) {
+        throw new Error("Error: denominator is 0!")
+    }
+    return numerator /denominator
```

Comment:
this would be more readable but your solution is fine

```js
function divideNumbers (numerator,denominator){
    if (typeof numerator !== 'number' || typeof denominator !== 'number'){
        throw new Error("Error: both parameters must be number!")
    }

    if(denominator === 0) {
        throw new Error("Error: denominator is 0!")
    }
    
    return numerator /denominator
}
```